tests/test_types.py

```diff
@@ -1170,3 +1171,42 @@ class Model(BaseModel):
             'type': 'type_error.integer',
         },
     ]
+
+
+def test_valid_simple_json():
+    class JsonModel(BaseModel):
+        json_obj: Json
+
+    obj = {'a': 1, 'b': [2, 3]}
```

Comment:
confused to dump then load. Better to use raw JSON strings in the tests

`'{"a": 1, "b": [2, 3]}'`

Reply:
and below.